### ISSUE TITLE
Add `TextChangedI` to list for `s:CheckEvent()`

### ISSUE THREAD

**Issue by Jorenar** [rickhowe/wrapwidth#2]

**Comment by rickhowe** [comment]
`TextChanged`is triggered when leaving the insert mode. `TechChangedI` is triggered each time a char is inserted so I avoided it.

**Comment by Jorenar** [comment]
Then perhaps it could be disabled by default, but user can enable it if desired?
Because it's really irritating when not only current "line" isn't wrapping as I type, but also "lines" below get/lose "indent"

**Comment by rickhowe** [comment]
I missed to check the `InsertLeave`  event. Please try version 1.3 or later. Still not possible to align the virtual spaces each time the char is typed.

**Comment by Jorenar** [comment]
With update 3.2 it's possible to do:
```vim
autocmd TextChangedI * if exists("b:wrapwidth") | Wrapwidth! | endif
```

**Comment by Jorenar** [comment]
Although such workaround isn't really working so well either

**Comment by Jorenar** [comment]
Fixed by #20abc78 (Update 3.3)
